client/src/main/java/io/pravega/client/batch/impl/BatchClientFactoryImpl.java

```diff
@@ -82,6 +88,7 @@ public class BatchClientFactoryImpl implements BatchClientFactory {
     private final Object lock = new Object();
     @GuardedBy("lock")
     private RawClient client = null;
+    private int retryAttempts = 10;
```

Comment:
final. (You'll need to reverse which constructor calls which)

Reply:
made the changes.